merkle/hstar2_test.go

```diff
@@ -25,6 +24,8 @@ import (
 	"github.com/google/trillian/testonly"
 )
 
+const treeID = int64(0)
```

Comment:
I'd suggest something other than zero for the ID. I'd also test that different treeID values result in different hashes.